src/hooks/useDatePicker.ts

```diff
@@ -0,0 +1,169 @@
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { Temporal } from "@js-temporal/polyfill";
+import { isoDateStringToZdt, zdtToIsoDateStr } from "../utils/conversions";
+import { useCalendarWeekDays } from "./internal/useCalendarWeekDays";
+import { useNavigation } from "./internal/useNavigation";
+import { useWeekDayLabels } from "./internal/useWeekDayLabels";
+import { SupportedNumberingSystem } from "../types";
+
+import "../date-override";
+import { isCustomCalendar } from "../utils/helpers";
+import calendarLocalisations from "../utils/calendarLocalisations";
+
+type DatePickerOptions = {
+  initialDate: string;
```

Comment:
This should be either `date` or `currentDate`. `initialDate` suggests tat when the value changes, nothing will happen, which is not true, there are hooks which will cause internal state changes